### ISSUE TITLE
Uses deprecated inspect.getargspec

### ISSUE THREAD

**Issue by dfrankow** [jjmontesl/cubetl#4]
```
  /app/cubetl/core/context.py:177: DeprecationWarning: inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()
    spec = getargspec(value)
```

I'm using python 3.7.4.

**Comment by jjmontesl** [comment]
Apparently we should use `inspect.signature` since Python 3.3. Let's go that way, as supporting older Python versions is not a requirement.

**Comment by jjmontesl** [comment]
Fixed in ecf0cb7 .
